podcast/2.5.md

```diff
@@ -0,0 +1,453 @@
+---
+title: 'The impact of connecting the right ideas with the right people, with Christopher Chedeau, creator of Excalidraw and co-creator of React Native'
```

Comment:
@csbailey5t @jennybergen Feel free to tweak this, but you get the idea!

Reply:
@rebeccadee we tweaked it a bit to help with the double "withs" in the title. This does move the name to the front of the title, and we should decide if we want to standardize the posts to have the content before the interviewee name or the interviewee's name first. If you feel strongly about this, feel free to change it tomorrow!

_Christopher Chedeau, creator of Excalidraw, co-creator of React Native on the impact of connecting the right ideas with the right people_